<launch>
  <param name="num_blocks" value="3" type="int" />
  <param name="configuration" value="scattered" type="str"/>
  <node name="sim_master" pkg="merlion_hw1" type="sim_master.py"/>
  <node name="controller" pkg="merlion_hw1" type="controller.py"/>
</launch>